bip-0001.mediawiki

```diff
@@ -67,7 +67,7 @@ Each BIP should have the following parts:
 
 * Abstract -- a short (~200 word) description of the technical issue being addressed.
 
-* Copyright/public domain -- Each BIP must either be explicitly labelled as placed in the public domain (see this BIP as an example) or licensed under the Open Publication License.
+* Copyright/public domain -- Each BIP must either be explicitly labelled as placed in the public domain (see this BIP as an example).
```

Comment:
@MarcoFalke I agree.

@luke-jr Heh yes, this is typical scope creep. @gmaxwell 's point here is to get rid of a license that is clearly not a good idea for this kind of thing. Enumerating which licenses **are** a good match for the BIP process is important, but maybe for another time.